### ISSUE TITLE
`--set-rec` seems to have no effect

### ISSUE THREAD

**Issue by SevereOverfl0w** [VladimirMarkelov/ttdl#57]
Reproduction, last commands shows that edit generally works.  It seems specific to rec that's broken.

```
/tmp 2m 17s
❯ ttdl --version                                               
TTDL Version 3.3.0

/tmp
❯ ttdl --todo-file todo.txt add 'Hello, world +filter'
Added todo:
# D P Created    Finished   Due        Subject
-----------------------------------------------
1     2022-07-24                       Hello, world +filter

/tmp
❯ ttdl --todo-file todo.txt ls +filter                
# D P Created    Finished   Due        Subject
-----------------------------------------------
1     2022-07-24                       Hello, world +filter
-----------------------------------------------
1 todos (of 1 total)

/tmp
❯ ttdl --todo-file todo.txt edit +filter --set-rec=1w
No todo was changed

/tmp
❯ ttdl --todo-file todo.txt edit +filter --set-proj=someproject 
Changed todos:
# D P Created    Finished   Due        Subject
-----------------------------------------------
1     2022-07-24                       Hello, world +filter +someproject
-----------------------------------------------
1 todos
```

**Comment by VladimirMarkelov** [comment]
Thanks for the report. I'll check it.

**Comment by VladimirMarkelov** [comment]
Fixed in `3.4.0` - just published

**Comment by SevereOverfl0w** [comment]
Works great, thanks!
